### PR TITLE
typo fix (it self -> itself)

### PR DESCRIPTION
**Pull request by adrinjalali**
also added "whereas" to show the contrast a bit clearer.

### CODE REVIEW

File: docs/reference/search/request/stored-fields.asciidoc [hunk 1]
```diff
@@ -40,7 +40,7 @@ GET /_search
 
 If the requested fields are not stored (`store` mapping set to `false`), they will be ignored.
 
-Field values fetched from the document it self are always returned as an array. Metadata fields like `_routing` and
+Field values fetched from the document itself are always returned as an array. Whereas metadata fields like `_routing` and
```
Comment (clintongormley) [inline docs/reference/search/request/stored-fields.asciidoc]: I'd change "Field values" to "Stored field values"

"Whereas" doesn't really work here - it should be ", whereas"  in this context.  That said, I prefer the sentence without whereas.  The intention is to make the contrast between the two sentences easy to spot.

Comment (adrinjalali) [inline docs/reference/search/request/stored-fields.asciidoc]: I agree with the "Stored field values". But don't you think a conjunctive would make the contrast clearer? My intention was also to bolden the contrast. If you don't lie whereas, maybe something like "on the contrary"?

Comment (clintongormley) [inline docs/reference/search/request/stored-fields.asciidoc]: "On the contrary" would work

**Comment by clintongormley** [comment]
thanks @adrinjalali - merged

**Pull request merged**